Kuzzle/API/Controllers/IndexController.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+
+namespace KuzzleSdk.API.Controllers {
+  /// <summary>
+  /// Implements the "index" Kuzzle API controller
+  /// </summary>
+  public class IndexController : BaseController {
+    internal IndexController(IKuzzleApi api) : base(api) { }
+
+    /// <summary>
+    /// Creates a new index in Kuzzle via the persistence engine.
+    /// </summary>
+    public async Task<JObject> CreateAsync(string index) {
+      Response response = await api.QueryAsync(new JObject {
+        { "controller", "index" },
+        { "action", "create" },
+        { "index", index },
+      });
+
+			return (JObject)response.Result;
```

Comment:
Your files are (very) badly indented.

Reply:
I forgot to install the VIM C# plugin :disappointed: